example/gmt/gmt_models.h

```diff
@@ -73,6 +73,18 @@ p4est_gmt_model_latlong_params_t;
 p4est_gmt_model_t  *p4est_gmt_model_latlong_new
   (p4est_gmt_model_latlong_params_t * params);
 
+/** Represents a segment of a geodesic in the sphere model.
```

Comment:
Would there be a simple way to have the doxygen parser include this .h file?

Reply:
We could add it to the file list. I think it makes sense to do this for the rest of the examples too. Currently the file list only tracks /src/, but I don't see why it shouldn't include all of /example/ as well.

Reply:
It is intentional not to have the example .c files in the list. They should be commented profusely with `/* */`.
For example .h files it's a different matters, and here I would agree to do this in a hand-picked way.

Reply:
The file list is set up to only parse .h files, so in theory we could just let it parse all the .h files in /example/. Or just hand pick ones to include.

There's also a separate examples list which doxygen gives its own page at /doc/html/examples.html. Currently the list is very outdated. I could add it to that list though and work on improving the list in the future.

Reply:
We might sit down with @SAutum to discuss how to go forward with documenting examples. This will be quite relevant down the road.